self-core-impl/src/test/java/com/selfxdsd/core/managers/StoredProjectManagerTestCase.java

```diff
@@ -1871,6 +1883,12 @@ public void handlesIssueLabelsChangesOfEstimation() {
         manager.issueLabelsChanged(event);
 
         Mockito.verify(task, Mockito.times(1)).updateEstimation(30);
+        final ArgumentCaptor<String> commentCaptor = ArgumentCaptor
+            .forClass(String.class);
+        Mockito.verify(comments, Mockito.times(1))
+            .post(commentCaptor.capture());
+        MatcherAssert.assertThat(commentCaptor.getValue(), Matchers
+            .startsWith("Estimation changed to ``30`` minutes at ``20"));
```

Comment:
//TODO we'll need to fix this test in 2100 :D

Reply:
took me a while to understand haha